nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/rs2d/RS2DData.java

```diff
@@ -162,7 +166,8 @@ public Dataset toDataset(String datasetName) throws IOException {
             dataset.setSw(i, getSW(i));
             dataset.setRefValue(i, getRef(i));
             dataset.setRefPt(i, dataset.getSizeReal(i) / 2.0);
-            dataset.setFreqDomain(i, true);
+            // State can be used to set the freq domain
+            dataset.setFreqDomain(i, getState(i) == 1);
```

Comment:
In Bruker, JCAMP and RS2D toDataset methods, the freq domain was just being set automatically to true. I changed here to use the state parameter to set the freq domain, but I am not sure if all processed spinit files will have this STATE parameter? If not maybe I should change back to default of true?

Reply:
They are supposed to, at least if a FT was done. If it isn't present, we can fallback to 0.

Reply:
Okay, the getState method does return 0 if STATE isn't present and has a log.debug message for that case as well